lib/services/database_test.go

```diff
@@ -189,3 +191,41 @@ func TestDatabaseFromRDSCluster(t *testing.T) {
 	require.NoError(t, err)
 	require.Equal(t, expected, actual)
 }
+
+func TestAuroraMySQLVersion(t *testing.T) {
+	require.Equal(t, "5.6.10a", auroraMySQLVersion(&rds.DBCluster{EngineVersion: aws.String("5.6.10a")}))
+	require.Equal(t, "1.22.1", auroraMySQLVersion(&rds.DBCluster{EngineVersion: aws.String("5.6.mysql_aurora.1.22.1")}))
+	require.Equal(t, "1.22.1.3", auroraMySQLVersion(&rds.DBCluster{EngineVersion: aws.String("5.6.mysql_aurora.1.22.1.3")}))
+}
+
+func TestIsRDSClusterSupported(t *testing.T) {
+	logger := logrus.WithField("test", t.Name())
+
+	tests := []struct {
+		name          string
+		engineMode    string
+		engineVersion string
+		isSupported   bool
+	}{
+		{name: "provisioned", engineMode: RDSEngineModeProvisioned, engineVersion: "5.6.mysql_aurora.1.22.0", isSupported: true},
+		{name: "serverless", engineMode: RDSEngineModeServerless, engineVersion: "5.6.mysql_aurora.1.22.0", isSupported: false},
+		{name: "parallel query supported", engineMode: RDSEngineModeParallelQuery, engineVersion: "5.6.mysql_aurora.1.22.0", isSupported: true},
+		{name: "parallel query unsupported", engineMode: RDSEngineModeParallelQuery, engineVersion: "5.6.mysql_aurora.1.19.6", isSupported: false},
```

Comment:
nit: Thanks for adding the keys. No big deal here, but it would be easier to read, with keys :), when formatted vertically. Most of the tests I've seen use a format like [this db test](https://github.com/gravitational/teleport/blob/18d27232f0838a609ff76db17f3885115464cae9/lib/srv/db/access_test.go#L67).

Reply:
Yeah, with long lines like this it read a bit better when it's formatted vertically.

Reply:
Thanks, will do